west.yml

```diff
@@ -43,7 +43,7 @@ manifest:
 
     - name: zephyr
       repo-path: zephyr
-      revision: d9c4ec31fc49e7eef3c8c3b0d07827cc04e6efee
+      revision: 5d902ea621e43ec9a3916eed8dea5ab3647d81c3
```

Comment:
There seems to be a regression in here.

Reply:
@softwarecki @abonislawski @juimonen @teburd At least locally Isee failures with the DMA start/stop changes. Here's a test run with Zephyr just before the DMA driver changes -> https://github.com/thesofproject/sof/pull/6998

Reply:
It was not that. The test run shows failures with HDMI before the start/stop changes on Zephyr side.

Reply:
@softwarecki @lgirdwood bisected problem to this change https://github.com/zephyrproject-rtos/zephyr/pull/53327#issuecomment-1406681383 -- this will show on MTL as well (once HDMI is enabled). I think we can't raise xruns on HDA link errors (to match previous behaviour expected on Linux).